sbin/build.sh

```diff
@@ -538,11 +538,12 @@ printJavaVersionString() {
        exit -1
      elif [ "${ARCHITECTURE}" == "riscv64" ]; then
        # riscv is cross compiled, so we cannot run it on the build system
-       # This is a temporary plausible solution in the absence of another fix
+       # So we fake it for now and retrive the version out from a downstream job on a riscv machine after the build
+       echo "Warning: java version can't be run on RISC-V build system. Faking version for now..."
        local jdkversion=$(getOpenJdkVersion)
        local jdkversionNoPrefix=${jdkversion#jdk-}
        local jdkShortVersion=${jdkversionNoPrefix%%+*}
-       cat << EOT > "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}/metadata/version.txt"
+       cat << EOT > "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}/metadata/fakedVersion.txt"
```

Comment:
I guess this can be removed now since it should never be required? Again, not critical and we can leave it in for now if it won't cause problems but perhaps a comment indicating it is no longer required as a minimum

Reply:
Done